shared/fs/really-delete/container.js

```diff
@@ -0,0 +1,44 @@
+// @flow
+import * as FsGen from '../../actions/fs-gen'
+import * as RouteTreeGen from '../../actions/route-tree-gen'
+import * as Container from '../../util/container'
+import ReallyDelete from '.'
+import {anyWaiting} from '../../constants/waiting'
+import * as Types from '../../constants/types/fs'
+import * as Constants from '../../constants/fs'
+
+type OwnProps = Container.RouteProps<{path: Types.Path}, {}>
+
+const mapStateToProps = state => {
+  return {
+    _deleting: anyWaiting(state, Constants.deleteWaitingKey),
+    title: 'Confirmation',
+  }
+}
+
+const mapDispatchToProps = dispatch => ({
```

Comment:
This actually accepts the `ownProps` as well:

```
const mapDispatchToProps = (dispatch, ownProps) => ({
...
```

So you can get the `path` here instead of passing from `mergeProps`.

Another thing to consider is, if `path` is not present in route props, we'd end up having the default path here, which is no fun to delete! So perhaps in the action here we should do something like `path !== Constants.defaultPath && dispatch(...`.